packages/web/src/components/collection/desktop/CollectionActionButtons.tsx

```diff
@@ -36,13 +38,32 @@ export const CollectionActionButtons = (props: CollectionActionButtonProps) => {
     isOwner,
     collectionId,
     onPlay,
+    onPreview,
     isPlaying,
     isPlayable,
+    isPreviewing,
     userId,
     tracksLoading,
     isPremium
   } = props
 
+  const { data: currentUserId } = useGetCurrentUserId({})
+  const { data: collection } = useGetPlaylistById(
+    {
+      playlistId: typeof collectionId === 'number' ? collectionId : -1,
```

Comment:
yeah this is a bit icky.
probably the audius-query thing should handle a null value and return nothing instead of this?
or wrap this in a HOC that would do something different on no collectionId?

Reply:
null idea seems good